server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -382,8 +435,18 @@ const accountMutations = {
 
       for (const key of Object.keys(args.account)) {
         switch (key) {
-          case 'currency':
+          case 'currency': {
+            models.Activity.create({
+              type: activities.COLLECTIVE_EDITED,
+              UserId: req.remoteUser.id,
+              UserTokenId: req.userToken?.id,
+              CollectiveId: account.id,
+              FromCollectiveId: account.id,
+              HostCollectiveId: account.approvedAt ? account.HostCollectiveId : null,
+              data: { previousData: { currency: account.currency }, newData: { currency: args.account[key] } },
+            });
```

Comment:
Can we do that after `setCurrency`, and only if the currency changes? `setCurrency` can fail, for example if there are transactions attached to the account

Reply:
Done. 👍🏽